source/cloud-security/amazon/services/supported-services/index.rst

```diff
@@ -55,6 +55,8 @@ The next table contains the most relevant information about configuring each ser
 +--------------+----------------------------------------------------------+-----------------------+----------------+------------------------------------------------------------------------------------------------------------------+
 | Amazon       | :ref:`Custom Logs Buckets <amazon_custom_logs>`          | subscriber            | buckets        |                                                                                                                  |
 +--------------+----------------------------------------------------------+-----------------------+----------------+------------------------------------------------------------------------------------------------------------------+
+| Amazon       | :ref:`Security Hub <amazon_security_hub>`                | subscriber            | security_hub   | <bucket_name>/<prefix>/<year>/<month>/<day>/<hh>                                                                 |
```

Comment:
On second thought, it might confuse the users since other services have a path to show the possibility of setting the `prefix` or `suffix` parameters. The objective of the path in the Security Hub integration was to provide a guide on where the logs could be found once every related AWS service was enabled.